### DISCUSSION TITLE
Feel free to hava a discussion here !

### DISCUSSION THREAD

**Discussion by YukinoshitaLove** [nkicsl/Resfusion#7] [General]
Below, I present some zero-shot results using Resfusion pretrained on LOL dataset.

## Daily life photos
![7BDD6E58798CF6417F5E16F567BA0FAF](https://github.com/user-attachments/assets/a9b94bc7-e936-4e16-b3b9-0493d71b2885)
![11CC2559A42465C9021215D85CC56426](https://github.com/user-attachments/assets/23e4aa18-9793-4780-b490-3bc619119bf5)

## Elden Ring
![微信图片_20241004114633](https://github.com/user-attachments/assets/76b9536c-830b-43f4-a2ea-eb67d53940ff)
![微信图片_20241004114624](https://github.com/user-attachments/assets/c6cadaec-8c72-4900-bb5b-a415039c304c)
![微信图片_20241004114639](https://github.com/user-attachments/assets/70b9b3c7-f1b2-4389-8a1a-37091c3f86de)

**Comment by YukinoshitaLove** [comment]
.ckpt files have been released. For details, see https://github.com/nkicsl/Resfusion/releases/tag/ckpts.

**Comment by wwxs123W** [comment]
hello！Are there checkpoints for image generation and image translation?

**Reply by YukinoshitaLove** [reply]
For image translation, you can simply replace the LQ image with the translated image. For image generation, set the LQ image to 0. The results should be easily reproducible following the settings in the appendix.
